code-challenges/set/Set.js

```diff
@@ -0,0 +1,69 @@
+class Set {
+  constructor(array = []) {
+    this.set = array;
+  }
+
+  add(value) {
+    if(!this.has(value)) this.set.push(value);
+  }
+
+  remove(value) {
+    for(let i = 0; i < this.set.length; i++) {
+      if(this.set[i] === value) {
+        this.set.splice(i, 1);
```

Comment:
after you find it you can break. Since sets only have 1 of any thing, you don't need to finish the for loop